td.vue/vue.config.js

```diff
@@ -30,13 +30,7 @@ const devServerConfig = hasTlsCredentials
         allowedHosts: [appHostname],
     }
     : {
-        client: {
```

Comment:
@ericfitz and @lreading , is this OK to remove? It was the cause of console errors for the desktop app - but seems a bit brutal of me just to delete it

Reply:
I believe it's fine to delete, unless @ericfitz disagrees! :)